porting/first-steps.rst

```diff
@@ -53,5 +53,19 @@ Install the required dependencies::
      python-markdown libxml2-utils xsltproc zlib1g-dev:i386 schedtool \
      repo liblz4-tool bc
 
+Arch
+^^^^^^^^^^
+
+If you have a pure ``amd64`` running you need to add ``[Multilib]`` repository to your ``/etc/pacman.conf`` . This will allow you to install and run ``i686`` packages. Please refer to `<https://wiki.archlinux.org/index.php/multilib>`_
+
+Install the required dependencies from official repository or AUR::
+
+   lineageos-devel git repo flex qemu binfmt-support qemu-user-static e2fsprogs sudo simg-tools
```

Comment:
i don't think that you need `qemu-user-static`. `qemu-arm-static` is the package required by @JBBgameich's script
also, you don't need QEMU at all i believe..

`repo` already depends on `git`, no need to manually specify it

`sudo` and `flex` are part of `base-devel` group, which should be assumed to be already installed by the user. Refer to AUR wiki for that

what is `binfmt-support`? it's not in repos or AUR, this package does not exist at all

`e2fsprogs` is part of `base` group which is populated when you install arch, it should not be there

`lineageos-devel` depends on `aosp-devel` which depends on `repo`. consider removing `repo` dependency then